File: archive/19-02/doc/js/res/code/terminal_settings.js
// Create settings with IP-address and port of terminal to connect to
let settings = new timapi.TerminalSettings();
settings.connectionIPString = "192.168.1.10";
settings.connectionIPPort = 80;

// Add wanted guides (with bitwise or)
settings.guides.add(timapi.constants.Guides.hospitality);

// Create terminal
let terminal = new timapi.Terminal(settings);

//...

// Disconnect from terminal and clean up properly
terminal.dispose();

File: archive/21-05/cs/res/code/terminal_settings.cs
using SIX.TimApi;

// Create settings with Terminal-ID of terminal to connect to
TerminalSettings settings = new TerminalSettings();
settings.TerminalId = "12345678";

// add wanted guides
settings.Guides.Add(Constants.Guides.Hospitality);
settings.Guides.Add(Constants.Guides.Gastro);

// Create terminal
Terminal terminal = new Terminal(settings);

//...

// Disconnect from terminal and clean up properly
terminal.dispose();
